app/tests/spec/views/confirm.js

```diff
@@ -164,7 +165,7 @@ define(function (require, exports, module) {
           callback();
         });
 
-        view.VERIFICATION_POLL_IN_MS = 100;
+        view.VERIFICATION_POLL_IN_MS = new Duration('100ms');
```

Comment:
@larissagaulia this needs to a `.milliseconds()`

Ref: https://tonicdev.com/vladikoff/56f8a8646f72571100d56ab7